packages/consent/consent-wrapper-onetrust/src/lib/onetrust-api.ts

```diff
@@ -21,9 +21,18 @@ type GroupInfoDto = {
 
 type OtConsentChangedEvent = CustomEvent<ConsentGroupIds>
 
+export enum OtConsentModel {
+  optIn = 'opt-in',
+  optOut = 'opt-out',
+  implicit = 'implied consent',
```

Comment:
Is `implicit` needed now? Doesn't look like it's exposed in `OneTrustSettings` and is currently an alias for `optIn`

Reply:
Since we only have unique behavior for opt-in and opt-out, I felt like it would be good to leave it out of the public API, but document that it exists in the enum, since this enum is meant to be an Internal DTO that maps to the actual OT API. I don't feel super strongly either way!